Seminars/Sem.04/Pract.04/Task02.cpp

```diff
@@ -0,0 +1,224 @@
+#include <iostream>
+#include <fstream>
+using std::cin;
+using std::cout;
+using std::endl;
+
+
+const int MAX_NAME_LENGTH = 20;
+const int CONDITION_LENGTH = 50;
+const int  NUMBER_OF_TASKS = 10;
+const int NUM_TASKS = 3;
+const int MIN_PASS_GRADE = 60;
+
+class Task {
+private:
+	char name[MAX_NAME_LENGTH];
+	char condition[CONDITION_LENGTH];
+	int points;
+public:
+	Task():name{}, condition{}, points{} {}
+	Task(const char* name, const char* condition, int points) {
+		setName(name);
+		setCondition(condition);
+		setPoints(points);
+	}
+	const char* getName()const {
+		return name;
+	}
+	const char* getCondition()const {
+		return condition;
+	}
+	int getPoints()const {
+		return points;
+	}
+
+	void setName(const char name[]) {
```

Comment:
It would be better to make this function private,because you use it in changename